test/index.js

```diff
@@ -50,3 +50,4 @@ require('./typedef');
 require('./const');
 require('./default');
 require('./enum');
+require('./unrecognized-exception');
```

Comment:
?

You mean this: https://www.npmjs.com/package/error ?

I thought we said it doesn't make sense to inherit from error (or at least it doesn't make sense to produce a stack trace). 

There is also a `require('./exception');` on L42. Should that also use the `error` module?